backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/RabbitMQConfiguration.java

```diff
@@ -488,6 +492,12 @@ public Builder hosts(Collection<Host> hosts) {
             return this;
         }
 
+        public Builder queueTTL(long ttl) {
+            Preconditions.checkArgument(ttl > 0, String.format("'%s' must be strictly positive", QUEUE_TTL));
```

Comment:
```suggestion
            Preconditions.checkArgument(ttl > 0, "'%s' must be strictly positive", QUEUE_TTL);
```

String formatting is built in checkArgument